schemas/v2/TextInputItem.json

```diff
@@ -353,6 +400,80 @@
         "yes"
       ]
     },
+    "TimeFormatOptions" : {
+      "$id" : "#TimeFormatOptions",
+      "type" : "object",
+      "title" : "TimeFormatOptions",
+      "description" : "",
+      "properties" : {
+        "allowFuture" : {
+          "type" : "boolean",
+          "description" : "Should this time range allow times that occur in the future? (ie. later today)"
```

Comment:
```suggestion
          "description" : "Should this time range allow times that occur in the future? (e.g. later today)"
```

Reply:
The abbreviation “i.e.” stands for id est, which is Latin for “that is” or “in other words.” The abbreviation “e.g.” stands for the Latin phrase exempli gratia, meaning “for example.” 

In this case, "later today" is not an example, it is a rephrasing of the question.